### PR TITLE
Task test coverage

### PR DESCRIPTION
**Pull request by adberglund**
* Removed tasks from the ignore list for coverage
* Altered and added tasks to call the task functions directly.
  * This allows for coverage to be measured on the tasks
  * This will cause test failures if we alter the tasks in breaking ways

The PR looks like more changes than it actually has. Because some code got shifted down it looks like a big chunk was added.

### CODE REVIEW

**Comment by adberglund** [comment]
I believe codecov uses the coveragerc in master to check the project and patch. So the patch coverage change is not accurately reflected here. Here's my local coverage

```
Name                                                          Stmts   Miss Branch BrPart  Cover   Missing
---------------------------------------------------------------------------------------------------------
masu/__init__.py                                                 31      1      4      1    94%   64, 63->64
masu/api/__init__.py                                              2      0      0      0   100%
masu/api/download.py                                             11      0      0      0   100%
masu/api/notification.py                                         26      0      4      0   100%
masu/api/status.py                                               49      0     14      1    98%   61->63
masu/celery/__init__.py                                          15      0      2      0   100%
masu/celery/tasks.py                                              8      0      0      0   100%
masu/database/__init__.py                                         2      0      0      0   100%
masu/database/auth_db_accessor.py                                13      0      0      0   100%
masu/database/customer_db_accessor.py                            19      0      0      0   100%
masu/database/koku_database_access.py                            37      0      0      0   100%
masu/database/provider_auth_db_accessor.py                       16      0      0      0   100%
masu/database/provider_billing_source_db_accessor.py             16      0      0      0   100%
masu/database/provider_collector.py                              15      0      2      0   100%
masu/database/provider_db_accessor.py                            51      0      0      0   100%
masu/database/report_db_accessor.py                             112      3     30      0    98%   233-235
masu/database/report_stats_db_accessor.py                        50      0     10      0   100%
masu/database/reporting_common_db_accessor.py                    25      0      6      0   100%
masu/exceptions.py                                                7      0      0      0   100%
masu/external/__init__.py                                         7      0      0      0   100%
masu/external/accounts/__init__.py                                0      0      0      0   100%
masu/external/accounts/cost_usage_report_account.py              18      0      0      0   100%
masu/external/accounts/db/__init__.py                             0      0      0      0   100%
masu/external/accounts/db/cur_accounts_db.py                     20      0      2      0   100%
masu/external/accounts_accessor.py                               16      0      4      0   100%
masu/external/downloader/__init__.py                              0      0      0      0   100%
masu/external/downloader/aws/__init__.py                          0      0      0      0   100%
masu/external/downloader/aws/aws_report_downloader.py            95      0     20      1    99%   80->79
masu/external/downloader/aws/utils.py                            44      0     10      0   100%
masu/external/downloader/report_downloader_base.py                7      0      2      0   100%
masu/external/notification_handler.py                            35      0     10      0   100%
masu/external/notifications/__init__.py                           0      0      0      0   100%
masu/external/notifications/aws/__init__.py                       0      0      0      0   100%
masu/external/notifications/aws/aws_notification_handler.py      86      0     26      0   100%
masu/external/report_downloader.py                               28      0      4      0   100%
masu/processor/__init__.py                                        5      0      0      0   100%
masu/processor/_tasks/__init__.py                                 0      0      0      0   100%
masu/processor/_tasks/download.py                                15      0      0      0   100%
masu/processor/_tasks/process.py                                 18      0      0      0   100%
masu/processor/orchestrator.py                                   29      0      8      0   100%
masu/processor/report_processor.py                              196      6     50      6    95%   144-150, 408, 441, 143->144, 152->155, 176->exit, 232->242, 407->408, 440->441
masu/processor/tasks.py                                          14      0      2      0   100%
masu/providers/__init__.py                                        0      0      0      0   100%
---------------------------------------------------------------------------------------------------------
TOTAL                                                          1138     10    210      9    99%
```

File: tests/processor/test_tasks.py [hunk 1]
```diff
@@ -78,62 +79,78 @@ def test_get_report_exception(self, fake_downloader):
...
+    """Test Cases for the Orchestrator object."""
+
+    @patch('masu.processor.report_processor.ReportProcessor')
+    @patch('masu.processor.report_processor.ReportProcessor.process', return_value=2)
+    def test_process_file(self, fake_processor, fake_process):
+        """Test task"""
+        request = {'report_path': '/test/path/file1.csv',
+                   'compression': 'gzip',
+                   'schema_name': 'testcustomer'}
+        _process_report_file(**request)
```
Comment (blentz) [inline tests/processor/test_tasks.py]: This test is missing an assertion.

Comment (adberglund) [reply tests/processor/test_tasks.py]: I had a merge conflict that I think I did not handle well. I'll get it back in.

Comment (adberglund) [reply tests/processor/test_tasks.py]: Ended up not being from the merge conflict. They were just missing. I believe this should cover it.

**Pull request merged**